contracts/MetadataViews.cdc

```diff
@@ -238,4 +238,30 @@ pub contract MetadataViews {
             self.url=url
         }
     }
+
+    // Edition is an optional view for collections that issues multiple objects
+    // with the same name, description and media file. This information is 
+    // useful for wallets and marketplaces.
+    // 
+    pub struct Edition {
+
+        // The edition number of the object.
+        //
+        // For an "24 of 100 (#24/100)" item, the editionNumber is 24. 
+        //
+        pub let editionNumber: UInt64
```

Comment:
We already know that the view is about editions, what's the reasoning for repeating that again with each variable name?